articles/implementing-flask-wtform/index.md

```diff
@@ -0,0 +1,407 @@
+
+This article requires your basic understanding of how to set up Flask web applications. If you do not know how to approach that, then read through my previous [article](https://www.section.io/engineering-education/complete-guide-on-installing-flask-for-beginners/).  Clone the GitHub [repository](https://github.com/corpsgeek/introduction-to-flask) to get started with implementing the Flask form.
+
+# Configuring The Work Environment
+Start by cloning the GitHub repository to your environment.
+
+```bash
+$ git clone https://github.com/corpsgeek/introduction-to-flask.git
+```
+Change the directory to the folder you have the GitHub repo cloned. Proceed to install the necessary package required for the cloned Flask application. Run the following command to install the required dependencies.
+
+```bash
+$ pip3 install -r requirements.txt
+```
+Note that if you are running Python2 only, then replace **pip3** with **pip**.
+
+## Possible Errors
+If you stumble upon the following error when installing the required dependencies for the cloned file, Ignore it and proceed to the next step.
+
+```bash
+$   Could not find a version that satisfies the requirement pkg-resources==0.0.0 (from -r requirements.txt (line 6)) (from versions: )
+No matching distribution found for pkg-resources==0.0.0 (from -r requirements.txt (line 6))
+```
+
+## Running The Application
+In light of any errors distinct from the one stated above, comment, and if not, proceed to run the flask application by executing the following in your terminal.
+
+```bash
+$ flask run
+```
+With this, you can proceed to your web browser and follow the URL stated in your terminal (127.0.0.1:500), and you should see a simple hello world application displayed.
+
+![Hello world image](/engineering-education/implementing-flask-wtform/hello_world.jpg)
+
+# Form Handling In Flask
+The Flask framework does not provide a unique way to handle forms. However, there are two ways developers go through the process of form handling and creation.
+
+## The Form Creation Technique
+The following are the two methods of creating forms in Flask:
+- HTML Forms
+- WTForms
+
+## The Form Handling Technique
+The following are the two methods of handling forms in Flask:
+- Request Object
+- Flask- WTF Extension
+
+# Form Creation With HTML Forms
+The first step to working with forms in your Flask application is to create your forms before handling them. You can create your forms using basic HTML and CSS. 
+
+However, for development and production purpose. The use of frameworks like Bootstrap because form error handling in Flask is easier.
+
+In our cloned Flask application, the HTML template is rendered from the app/views.py file, displaying a Hello World text. Now, we would undergo some coding sprint here to structure this application properly.
```

Comment:
Rephrase the second sentence. We would undergo.... -> We use the above files to code the application.